client/my-sites/plans-features-main/hooks/use-generate-action-callback.ts

```diff
@@ -78,7 +78,8 @@ function useUpgradeHandler( {
 					product_slug: selectedStorageAddOn.productSlug,
 					quantity: selectedStorageAddOn.quantity,
 					volume: 1,
-					extra: { feature_slug: selectedStorageAddOn?.featureSlugs?.[ 0 ] },
+					// TODO: `feature_slug` below should be refactored to `add_on_slug` now that the two concepts are separated
+					extra: { feature_slug: selectedStorageAddOn?.addOnSlug },
```

Comment:
We need to investigate the use of `feature_slug` here and how to update to `add_on_slug`. There are other similar instances in the code that will need to be accounted for.

Reply:
Issue https://github.com/Automattic/wp-calypso/issues/92373